test/AutoRest.TestServer.Tests/Mgmt/TestProjects/TestProjectTests.cs

```diff
@@ -90,5 +89,104 @@ private IEnumerable<Type> FindAllContainers()
                 }
             }
         }
+
+        private IEnumerable<Type> FindAllRestOperations()
+        {
+            Type[] allTypes = Assembly.GetExecutingAssembly().GetTypes();
```

Comment:
I think this could just be:

```
private IEnumerable<Type> FindAllRestOperations() => Assembly.GetExecutingAssembly().GetTypes().Where (t => t.Name.Contains("RestOperations") && !t.Name.Contains("Tests") && t.Namespace == _projectName); 
```

If you want to lean more on LINQ.

Reply:
I think this is more readable. That's why I didn't use the LINQ here.